src/app/service/tv-show.service.ts

```diff
@@ -0,0 +1,34 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { ITvShow } from '../interface/tv-show';
+import { Observable, map } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class TvShowService {
+  defaultSearchUrl = 'https://www.episodate.com/api/search';
+
+  constructor(private http: HttpClient) { }
+
+  getTvShows(name: string = ''): Observable<ITvShow[]> {
+    const url = `https://www.episodate.com/api/search?q=${name}&page=1`
+
+    return this.http.get<any>(url).pipe(
```

Comment:
I always recommend [avoiding any in your code](https://www.angulartraining.com/daily-newsletter/typescript-any-vs-unknown/)

Reply:
I have fixed my interface and added its type to this. Thank you.